.travis.yml

```diff
@@ -31,7 +31,10 @@ install:
 script:
   - if [[ $TRAVIS_RUST_VERSION == "stable" && $TRAVIS_OS_NAME == "linux" ]]; then cargo fmt -- --check; fi
   - if [[ $TRAVIS_RUST_VERSION == "stable" && $TRAVIS_OS_NAME == "linux" ]]; then cargo clippy -- -D clippy::all; fi
+  - if [[ $TRAVIS_RUST_VERSION == "stable" && $TRAVIS_OS_NAME == "linux" ]]; then cargo clippy --no-default-features --features prost-codec -- -D clippy::all; fi
   - cargo test --all -- --nocapture
+  # There is a bug in protobuf-build that cached size is not supported yet.
```

Comment:
Is the bug still here?

Reply:
Yes. It can be fixed later. Making rust-protobuf back is with highest priority.